docs/document_en/website/pages/en/versions.js

```diff
@@ -86,19 +86,18 @@ const Versions = (props) => {
             </tbody>
           </table>
           <p>
-            ※ The version of the v0 system is
+            ※ Please find the version of v0 system
             <a
               href="https://kintone-labs.github.io/kintone-ui-component/latest/versions/"
               target="_blank"
               rel="noreferrer"
             >
               Here
             </a>
-            Confirm.
           </p>
           <h3 id="rc">Pre-release versions</h3>
           <p>
-            You can check the latest changes which have not yet been released from the following section.
+            You can check the latest changes which yet to be released from the following section.
```

Comment:
> __[ESLint]__ Insert `⏎···········` ([view](https://sider.review/gh/repos/133498505/pulls/521#issue-5579951))
>
> <details>
>
> | Rule | Severity | Category | Recommended |
> | ---- | -------- | -------- | ----------- |
> | `prettier/prettier` | `error` | - | - |
>
> References:
> - <https://github.com/prettier/eslint-plugin-prettier#options>
>
> You can [close this issue](https://sider.review/gh/repos/133498505/pulls/521#issue-5579951) if no need to fix it. [Learn more](https://help.sider.review/getting-started/working-with-issues).
>
> </details>

You can check the latest changes which are scheduled for releasing in the following section.